apps/src/netsim/DashboardUser.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2015 Code.org
+ * http://code.org/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Interface to dashboard user data API.
+ */
+
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
```

Comment:
Because this might be unfamiliar: This is a stricter JSHint configuration than our defaults, that's used only for the current file.  I find it keeps me honest.  Hopefully some of these will eventually move into our global JSHint config.

Check out the [list of JSHint options](http://jshint.com/docs/options/) if you're curious.

I use the `global` directive here because I noticed Brent started making comments about what globals were being used in each file.  Doing it this way, JSHint actually reads and enforces the annotation to keep it accurate, complaining of undeclared globals or unused global declarations.